src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -104,6 +83,7 @@ void buildHearingDetails() {
                                         .baseLocation(EPIMS_ID)
                                         .region(REGION)
                                         .build())
+            .dwpIsOfficerAttending("Yes")
```

Comment:
Where did these two changes come from?

Reply:
Quick fix to some tests so they don't fall into the isPaperCase() path of new code.